client/App.jsx

```diff
@@ -10,6 +10,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      name: 'Morning Joggers',
```

Comment:
We'll need to move the api calls to App later to ensure you can pass the name down as props